<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_14">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="HashTextInputButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>打开</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="HashEncodingTips">
           <property name="text">
            <string>编码表:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="HashEncodingBox">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="HashSaltTips">
           <property name="text">
            <string>加盐:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="HashSaltBox">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="HashEvalCheckBox">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>Eval</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="HashEncodeButton">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>45</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>45</height>
            </size>
           </property>
           <property name="text">
            <string>计算</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextEdit" name="HashTextBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>516</width>
          <height>731</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="HashTips">
             <property name="text">
              <string>Hash:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="HashValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="MD5Tips">
             <property name="text">
              <string>MD5:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="MD5ValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="SHA1Tips">
             <property name="text">
              <string>SHA1:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SHA1ValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="SHA224Tips">
             <property name="text">
              <string>SHA224:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SHA224ValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="SHA256Tips">
             <property name="text">
              <string>SHA256:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SHA256ValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="SHA384Tips">
             <property name="text">
              <string>SHA384:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SHA384ValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="SHA512Tips">
             <property name="text">
              <string>SHA512:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SHA512ValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="SHA3224Tips">
             <property name="text">
              <string>SHA3-224:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SHA3224ValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="SHA3256Tips">
             <property name="text">
              <string>SHA3-256:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SHA3256ValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="SHA3384Tips">
             <property name="text">
              <string>SHA3-384:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SHA3384ValueBox"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="SHA3512Tips">
             <property name="text">
              <string>SHA3-512:</string>
             </property>
             <property name="textFormat">
              <enum>Qt::AutoText</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="SHA3512ValueBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
